cmd/release-controller-api/http.go

```diff
@@ -563,7 +586,11 @@ func (c *Controller) getReleaseTagInfo(req *http.Request) (*releaseTagInfo, erro
 
 func (c *Controller) httpReleaseInfo(w http.ResponseWriter, req *http.Request) {
 	start := time.Now()
-	defer func() { klog.V(4).Infof("rendered in %s", time.Now().Sub(start)) }()
+	defer func() { klog.V(4).Infof("rendered in %s", time.Since(start)) }()
+
+	if endOfLifePrefixes == nil || req.URL.Query().Get("forceRefresh") == "true" {
+		c.endOfLifePrefixes()
+	}
```

Comment:
I can't image that anyone, or anything, would ever call this API with the `forceRefresh` option specified, so this list will inevitably be stale after it's initial population.
How much time does it add to the call if we refresh the list each time?

Reply:
It takes about 2.115695ms. I added the `forceRefesh` option mainly because I missread the time initially (I thought it was 2 seconds, but it is `ms`).
Shall I remove it? 

Alternatively, I can add a force refresh button in the UI, to make it more accessible.

Reply:
Done